float32 x
float32 y
float32 z
#yaw is in degree from -180 to 180
float32 yaw
---
bool result 
